tests/unit/test_landing_pad_tracking.py

```diff
@@ -124,9 +124,12 @@ def test_is_similar_positive_equal_to_threshold(self) -> None:
         them is equal to the distance threshold.
         """
         obj_1 = object_in_world.ObjectInWorld.create(0, 0, 0)[1]
+        assert obj_1 is not None
```

Comment:
we want to have it like this:
result, obj_1 = ...
assert result

Reply:
Nope, that's all I found